college_preprocessor.py

```diff
@@ -6,81 +6,180 @@
         if int(begin) < 100:
             return begin
 
-    return 'None'
+    return 0
 
-def ACT2(score):
+def ACT_end(score):
     '''
     Splits the ACT score range string into two real valued numbers
     '''
-    if score != 'None':
+    if isinstance(score, basestring):
         if '-' not in score:
             end = score
         else:
             end = score.split('-')[1]
         if int(end) < 100:
             return end
 
-    return 'None'
+    return 0
 
 def ratio(rat):
     '''
     Converts the student-faculty ratio string into a real valued number
     '''
-    if rat != 'None':
+    if isinstance(rat, basestring):
         student = rat.split(':')[0]
         faculty = rat.split(':')[1]
         return float(student)/float(faculty)
-    return 'None'
+    return 0
+
+def strip_chars(value):
+    '''
+    Strips all cell values of ,$% for easy parsing
+    '''
+    strvalue = value
+    if isinstance(strvalue, basestring):
+        strvalue = strvalue.replace('$', '')
+        strvalue = strvalue.replace(',', '')
+        if '%' in strvalue:
+            strvalue = strvalue.replace('%', '')
+            strvalue = str(float(strvalue)/100)
+        return strvalue
+    else:
+        return value
+
+def separate_parens(value):
+    '''
+    Some cells have an unecessary description within parenthesis,
+    so just remove it from the cell
+    '''
+    strvalue = value
+    if isinstance(strvalue, basestring):
+        if '(' in strvalue:
+            strvalue = strvalue.split('(')
+            return strvalue[0]
+    
+    return value
+
+def category_yn(value):
+	'''
+	Cells with yes/no values are changed to 1/0
+	yes = 1
+	no = 0
+	'''
+	if isinstance(value, basestring):
+		if value == 'Yes':
+			return 1
+		if value == 'No':
+			return 0
+	return value
+
+def category_enlist(value):
+	'''
+	Category parsing for army/navy/airforce
+	not offered = 0
+	offered on campus = 1
+	offered at other institute = 2
+	'''
+	if isinstance(value, basestring):
+		if 'Offered on campus' in value:
+			return 1
+		if 'Offered at cooperating institution' in value:
+			return 2
+	return 0
+
+def category_tests(value):
+	'''
+	Category parsing for standardized tests
+	No tests required = 0
+	SAT only = 1
+	ACT only = 2
+	EIther SAT or ACT = 3
+	'''
+	if isinstance(value, basestring):
+		if value == 'Neither SAT nor ACT':
+			return 0
+		if value == 'SAT':
+			return 1
+		if value == 'ACT':
+			return 2
+		if value == 'Either SAT or ACT':
+			return 3
+	return 0
+
+def category_selectivity(value):
+	'''
+	Category parsing for selectivity
+	Least selective = 0
+	Less selective = 1
+	Selective = 2
+	More selective = 3
+	Most selective = 4
+	'''
+	if isinstance(value, basestring):
+		if value == 'Least selective':
+			return 0
+		if value == 'Less selective':
+			return 1
+		if value == 'Selective':
+			return 2
+		if value == 'More selective':
+			return 3
+		if value == 'Most selective':
+			return 4
+	return 0
```

Comment:
Does not having a value that matches the choices mean that the school is 'least selective' or that it wasn't included. If it wasn't included, we should probably modify the default to be a unique number like -1 or shift the other values up one value.